client/app/components/parameter-settings.html

```diff
@@ -13,6 +13,7 @@ <h4 class="modal-title">{{$ctrl.parameter.name}}</h4>
         <select ng-model="$ctrl.parameter.type" class="form-control">
           <option value="text">Text</option>
           <option value="number">Number</option>
+          <option value="enum">Enum</option>
```

Comment:
Maybe not often, but they do and we should be friendly :-)

How about "Dropdown List"?